### ISSUE TITLE
Monodevelop for Ubuntu 20.04 LTS

### ISSUE THREAD

**Issue by UltimateYZ** [mono/monodevelop#9638]
Hi mono developers,

I wanted to know when the MonoDevelop version for Ubuntu 20.04 LTS will be released. The current version has too many bugs and is missing many GUI form components compared to glade or Visual Studio Designer. And finally, GTK# 2 is very old, when GTK# 3 will be integrated in monodevelop designer?

**Comment by scherenhaenden** [comment]
There are not new releases for the current versions of Ubuntu... and you are asking for 20.04 :s?

**Comment by undera** [comment]
Yes, since 20.04 is LTS release of Ubuntu, many people (including me) are migrating onto it. So it becomes crucial to get MonoDevelop support for Ubuntu 20.04

**Comment by scherenhaenden** [comment]
I feel you guys, I have the same problem. I'm using Monodevelop and Rider right now, I did not migrate till now and I'm even thinking of chaning to Java... Fact is, that Monodevelop is not being developed to be used in Linux, since MS is not interested

**Comment by ryhlyj-tvorog** [comment]
> I feel you guys, I have the same problem. I'm using Monodevelop and Rider right now, I did not migrate till now and I'm even thinking of chaning to Java... Fact is, that Monodevelop is not being developed to be used in Linux, since MS is not interested

Yes. Unfortunately it seems so. But why Java? Maybe Kotlin then?

**Comment by scherenhaenden** [comment]
Because in my new job I'm working with Java. XD!

**Comment by lextm** [comment]
https://github.com/dotdevelop/dotdevelop/tree/dotdevelop_oe_8.4.3.12_NoGit

You can build it on Ubuntu 20.04 now. The GitHub Actions config is at https://github.com/dotdevelop/dotdevelop/blob/dotdevelop_oe_8.4.3.12_NoGit/.github/workflows/monodevelop.yml
